.github/workflows/dependency-update.yml

```
@@ -0,0 +1,52 @@
+        run: |
+          npm update
+          npm audit fix || true
+
+      - name: Update Python dependencies
+        if: hashFiles('requirements.txt') != ''
+        run: |
+          pip install --upgrade pip
+          pip install -r requirements.txt --upgrade
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)